website/src/components/FooterComponent.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ListView from "./ListView";
+const copyright = `Copyright © ${new Date().getFullYear()}`;
+
+
+const FooterComponent = ({ footerData }) => {
+    return (
+        <footer className="footer-component">
+            <div className="container container-fluid">
+                <div className="footer_copy">
+                    <a href="https://imply.io/#"><img src="/img/logo.png" /></a>
+                    <p>Imply, the real-time analytics platform built from Apache Druid, is a fast, easy way to provide the best experience possible for your analytics apps.</p>
+                    <a href="https://imply.io/#">Read more about Apache Druid</a>
+                </div>
+                <div className="footer__links">
+                    {Object.keys(footerData).map((key, index) => (
+                        <ListView key={index} listData={footerData[key].content} listDataHeader={footerData[key].header} />
+                    ))}
+                </div>
+            </div>
+            <div className="container container-fluid">
+                <div className="socials-container">
+                    <a href="https://www.linkedin.com/company/imply/" target="_blank"><img src="/img/linkedin.png" /></a>
+                    <a href="https://twitter.com/implydata" target="_blank"><img src="/img/twitter.png" /></a>
+                    <a href="https://www.youtube.com/c/Implydata" target="_blank"><img src="/img/youtube.png" /></a>
+                    <a href="https://github.com/apache/druid-website-src" target="_blank"><img src="/img/github.png" /></a>
```

Comment:
Website src is a weird locale to link to for this site? Maybe just implydata/learn-druid